OracleTuxedo/art_runtime/README.md

```diff
@@ -0,0 +1,35 @@
+# Introduction
+This folder contains the information and examples of how to use [Tuxedo ART Runtime](http://docs.oracle.com/cd/E72452_01/artrt/docs1222/index.html) with [Docker](https://www.docker.com/).
+
+## Contents
+It is based on the Oracle Tuxedo image.
+
+##Prerequisite
+1. Use btrfs with docker:
+   With big image size, it's better to use btrfs with docker, please follow the link below to set Docker using btrfs.
+   https://docs.docker.com/engine/userguide/storagedriver/btrfs-driver/#configure-btrfs-on-sles
```

Comment:
This points to SLES documentation on an external site, which is not acceptable. 

If you must link to documentation, please use the [Oracle Linux Docker User's Guide](http://docs.oracle.com/cd/E52668_01/E87205/html/index.html) instead.